backend/battling/views.py

```diff
@@ -51,25 +49,56 @@ def form_valid(self, form):
         else:
             messages.success(self.request, "Battle ended! Check e-mail for results.")
 
-            # run_battle_and_send_email(battle)
+            get_battle_winner(battle)
 
         return super().form_valid(form)
 
 
-class DeleteTeam(DeleteView):
-    template_name = "battling/delete_team.html"
+class DeleteBattle(DeleteView):
+    template_name = "battling/delete_battle.html"
     success_url = reverse_lazy("home")
-    queryset = Team.objects.all()
+    queryset = Battle.objects.all()
 
     def get_success_url(self):
         messages.success(self.request, "Battle refused!")
 
         return reverse_lazy("home")
 
 
+class SettledBattles(ListView):  # pylint: disable=too-many-ancestors
+    template_name = "battling/settled_battles.html"
+    model = Battle
+
+    queryset = Battle.objects.filter(status="SETTLED")
```

Comment:
Here there's no filter for current user. Are we tackling this later?

Reply:
No, because I show all the settled battles whether the user created or was invited, but I can change it to make it like the ongoing battles

Reply:
But maybe you can also see here stuff from other users?